source/_integrations/satel_integra.markdown

```diff
@@ -40,133 +40,37 @@ A list of all partition, zone and output IDs can be acquired by running DloadX p
-  description: "This parameter lists the zones (or inputs) that will be visible by Home Assistant. For each zone, a proper ID must be given as well as its name. The name is arbitrary and does not need to match the one specified in Satel Integra alarm configuration."
-  required: false
-  type: [integer, list]
-  keys:
-    name:
-      description: Name of the zone.
-      required: true
-      type: string
-    type:
-      description: The zone type.
-      required: false
-      default: motion
-      type: string
-outputs:
-  description: "Very similar to zones, but with outputs. Satel Integra uses outputs to inform external systems about different events. For example power failure, or that alarm started counting for exit or some other user-defined condition. They may be used for simple alarm-based automation. For more information please refer to Satel homepage and forums."
-  required: false
-  type: [integer, list]
-  keys:
-    name:
-      description: Name of the output.
-      required: true
-      type: string
-    type:
-      description: The type of the device - just for presentation.
-      required: false
-      default: motion
-      type: string
-switchable_outputs:
-  description: "Switchable outputs. These will show up as switches within Home Assistant."
-  required: false
-  type: [integer, list]
-  keys:
-    name:
-      description: Name of the output.
-      required: true
-      type: string
-{% endconfiguration %}
-
-## Full examples
+After setting up the connection details, partitions, zones and outputs can be configured using the <i>Options</i> on the <i>Satel Integra</i> card on the {% my integrations title="Settings > Devices & services" %} page.
 
-```yaml
-# Example configuration.yaml entry
-satel_integra:
-  host: 192.168.1.100
-  port: 7094
-  partitions:
-    01:
-      name: "House"
-      arm_home_mode: 2
-    02:
-      name: "Garage"
-  zones:
-    01:
-      name: "Bedroom"
-      type: "motion"
-    02:
-      name: "Hall"
-      type: "motion"
-    30:
-      name: "Kitchen - smoke"
-      type: "smoke"
-    113:
-      name: "Entry door"
-      type: "opening"
-  outputs:
-    05:
-      name: "Garden lights trigger"
-      type: "light"
-    09:
-      name: "Gate opening trigger"
-      type: "opening"
-    30:
-      name: "Alarm triggered"
-      type: "safety"
-    32:
-      name: "Alarm power problem"
-      type: "safety"
-  switchable_outputs:
-    05:
-      name: "Gate open"
-    06:
-      name: "Gate close"    
-    14:
-      name: "Garden light"
-      
-```
+### Partitions, Zones and Outputs
+
+Partitions, Zones and outputs can be added, edited, and removed through the option forms.
+
+Each partition will have it's own alarm panel, each zone and output will have a binary sensor and each switchable output will have a switch created.
+
+To get started, select which entry type you want to manage:
+
+![image](https://github.com/user-attachments/assets/c788689b-9f58-46c4-9e51-d6cce5431763)
```

Comment:
please upload the images into this repo, under `/images/integrations/satel_integra`